File: 天梯赛/java/L1-031. 到底是不是太胖了.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		int n=in.nextInt();
		while(n!=0) {
			n--;
			int h=in.nextInt(),w=in.nextInt();
			double wei=(h-100)*0.9*2;
			if(Math.abs(w*1.0-wei)<wei*0.1)System.out.println("You are wan mei!");
			else if((w*1.0-wei)>=wei*0.1)System.out.println("You are tai pang le!");
			else System.out.println("You are tai shou le!");
		}
		in.close();
	}
}

File: 天梯赛/L1-031. 到底是不是太胖了.cpp
#include<iostream>
#include<cstring>
#include<vector>
#include<string>
#include<cstring>
#include<queue>
#include<set>
#include<stack>
#include<map>
#include<cmath>
#include<cstdio>
#include<algorithm>
using namespace std;
#define lowbit(i) ((i)&(-i))
typedef long long ll;
const int maxn=1e5+10;
const int inf=0x3f3f3f3f;

int main(){
	//ios::sync_with_stdio(false);
	int n;
	cin >> n;
	for(int i = 0; i < n; i++) {
		float height, weight;
		cin >> height >> weight;
		float stan = (height - 100) * 0.9 * 2;
		if(abs(stan - weight) < stan * 0.1) {
			cout << "You are wan mei!" << endl;
		} else if(stan > weight) {
			cout << "You are tai shou le!" << endl;
		} else {
			cout << "You are tai pang le!" << endl;
		}
	}
	return 0;
}
